sriovnet_switchdev.go

```diff
@@ -202,11 +202,11 @@ func GetVfRepresentorSmartNIC(pfID, vfIndex string) (string, error) {
 	// Note: no supoport for Multi-Chassis Smart-NICs
 	expectedPhysPortNames := map[string]interface{}{
 		fmt.Sprintf("pf%svf%s", pfID, vfIndex):   nil,
-		fmt.Sprintf("c0pf%svf%s", pfID, vfIndex): nil,
+		fmt.Sprintf("c1pf%svf%s", pfID, vfIndex): nil,
 	}
 
 	netdev, err := findNetdevWithPortNameCriteria(func(portName string) bool {
-		// if phys port name == pf<pfIndex>vf<vfIndex> or c0pf<pfIndex>vf<vfIndex> we have a match
+		// if phys port name == pf<pfIndex>vf<vfIndex> or c1pf<pfIndex>vf<vfIndex> we have a match
```

Comment:
in case of multihost, controller id can be 1,2,3,4.
so if you are looking for a phys_port_name, have controller number as search option.

Reply:
At this point we just want to solve the single host use-case.

Reply:
ok,